download-test-tool-from-artifactory.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -eux
+
+artifactory_name=matching-service-test-tool
+server_location=https://artifactory.ida.digital.cabinet-office.gov.uk/artifactory/remote-repos/uk/gov/verify
+version=`curl -s $server_location/$artifactory_name/maven-metadata.xml | grep release | sed "s/.*<release>\([^<]*\)<\/release>.*/\1/"`
+zip_name=$artifactory_name-$version.zip
+
+url="$server_location/$artifactory_name/$version/$zip_name"
+
+echo "Downloading the test tool zip file..."
+curl -O $url
+
+unzip matching-service-test-tool-*.zip
```

Comment:
can the wildcard be replaced with $version?

Reply:
Good catch! I have changed it now.